src/components/ScreenStackItem.tsx

```diff
@@ -74,7 +77,13 @@ function ScreenStackItem({
        * See https://github.com/software-mansion/react-native-screens/pull/1825
        * for detailed explanation.
        */}
-      <ScreenStackHeaderConfig {...headerConfig} />
+      <ScreenStackHeaderConfig
+        {...headerConfig}
+        hidden={
+          headerConfig?.hidden ||
+          (stackPresentation === 'formSheet' && Platform.OS === 'android')
```

Comment:
What does this change do?

Reply:
We want to ensure the header is not considered when layouting android formSheet, since it doesn't have one. It prevents sheet content height from being reduced by header height, like so:

![Screenshot_1730203067](https://github.com/user-attachments/assets/c9519dcf-7ccd-4662-9b59-02191790b475)

Reply:
It'd be good to add a comment here

Reply:
https://github.com/software-mansion/react-native-screens/pull/2462/commits/b110017917e3bc6e76e94d9d036c5305d5a085a5